### ISSUE TITLE
To have multiple floor more than 2 floors

### ISSUE THREAD

**Issue by allangan85** [ucdavis/wayfinding#30]
Hi can we have more than 2 floors like floor 3 or floor 4 for the routing to be working?

**Comment by skyrbe** [comment]
Hi Allangan85, Yes, I believe so. You can have any number of floors, you only need to make sure that the portals, doors and paths are correct in your SVGs.

**Comment by allangan85** [comment]
Hi Skyrbe, thank you for your explain. Yes it can straight go to the portal. But now let say we need to go from floor 1 to floor 3, and at the middle it will pass some path in floor 2 is it possible?
Like it will show path from Floor 1 to Floor 2 then last to Floor 3.

**Comment by skyrbe** [comment]
Hi allangan85, yes, it is possible. Your portals are connected to one another. For example, floor0 is connected to floor1 via a portal named floor1Stair0 , and floor1 and floor2 are connected via a portal named floor2Stair1. The wayfinding algorithm traverses from your location to the first floor and from there to the second floor. All you need to do is that the paths are connected properly,and you don't have a portal directly between floor 0 and floor 2.

**Comment by allangan85** [comment]
Yes, we tried it and it is working perfectly! Your script is fantastic and saving my life! Appreciate it!

**Comment by skyrbe** [comment]
Glad that it worked for you! I am just an interested 3rd party! Kudos to the dev team here! :-)

**Comment by allangan85** [comment]
Yes thanks for the UC Davis and dev team!

**Comment by hemantjha28** [comment]
Hi thanks for the great piece of work, but I am somehow unable to integrate more than 2 floors. I tried to comprehend the explanation given by skyrbe. Is there any sample code to integrate multiple floors. Thanks Again

**Comment by skyrbe** [comment]
@hemantjha28 It would help if you can share the SVG that you have for the floor plan. A lot of things are done via the SVG.

**Comment by hemantjha28** [comment]
@skyrbe . Thanks for your reply. I am able to manage it.

**Comment by skyrbe** [comment]
@hemantjha glad to know.
